counsel.el

```diff
@@ -5087,6 +5088,30 @@ this if you want to provide specific non-standard build types to
 `counsel-compile'.  The default helpers are setup to handle common
 build environments.")
 
+;; Should this be hook-able to deal with project managers and other
+;; version control systems?
```

Comment:
Yes.  I don't like each package to reinvent the wheel about this.  This has been implemented at least a dozen of times already.